public/js/Filter.js

```diff
@@ -1,23 +1,26 @@
-function SampleFilter(uniqueRow, uniqueCol, uniquePlate, uniqueGene) {
+var _ = require('lodash');
+
+function SampleFilter(sampleData) {
```

Comment:
I just had a look at the syntax myself. Looks good to me! I was more thinking there's no use *parsing* the doc strings as the UI methods won't be documented outside the source code.